### PR TITLE
deleted unnecessary arguments on ERC1155

### PR DESCRIPTION
**Pull request by victor-ego**
### What I did
deleted unnecessary arguments that were not being used on ERC1155.vy also modified test_erc1155.py arguments for testing

### How I did it
deleted them from the functions
### Commit message

Commit message for the final, squashed PR. (Optional, but reviewers will appreciate it! Please see [our commit message style guide](../../blob/master/docs/style-guide.rst#best-practices-1) for what we would ideally like to see in a commit message.)

### Cute Animal Picture

Put a link to a cute animal picture inside the parenthesis-->

### CODE REVIEW

File: examples/tokens/ERC1155ownable.vy [hunk 1]
```diff
@@ -297,7 +297,7 @@ def setApprovalForAll(owner: address, operator: address, approved: bool):
     log ApprovalForAll(owner, operator, approved)
 
 @external
-def safeTransferFrom(sender: address, receiver: address, id: uint256, amount: uint256, bytes: bytes32):
```
Comment (fubuloubu) [inline examples/tokens/ERC1155ownable.vy:300]: The standard has a `data: Bytes` field here

File: examples/tokens/ERC1155ownable.vy [hunk 2]
```diff
@@ -316,7 +316,7 @@ def safeTransferFrom(sender: address, receiver: address, id: uint256, amount: ui
     log TransferSingle(operator, sender, receiver, id, amount)
 
 @external
-def safeBatchTransferFrom(sender: address, receiver: address, ids: DynArray[uint256, BATCH_SIZE], amounts: DynArray[uint256, BATCH_SIZE], _bytes: bytes32):
```
Comment (fubuloubu) [inline examples/tokens/ERC1155ownable.vy:319]: The standard has a `data: Bytes` field here

File: tests/.gitignore [hunk 3]
```diff
@@ -0,0 +1,6 @@
+__pycache__
```
Comment (fubuloubu) [inline tests/.gitignore:1]: Could you remove all of the additional files you added from the PR?

**Comment by victor-ego** [comment]
included comments from @fubuloubu regarding ERC1155 standard arguments `bytes: bytes32` and unnecessary files uploaded on test folder

**Comment by fubuloubu** [comment]
> included comments from @fubuloubu regarding ERC1155 standard arguments `bytes: bytes32` and unnecessary files uploaded on test folder

still see the files

**Review by fubuloubu — APPROVED** [review]

**Comment by fubuloubu** [comment]
If you delete those three empty files we can merge

**Comment by victor-ego** [comment]
deleted empty files @fubuloubu I'm ready to merge 👉👈

**Review by fubuloubu — APPROVED** [review]

**Pull request merged**